sequel-ace.xcodeproj/project.pbxproj

```diff
@@ -2914,31 +2903,6 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		1A47995F2543320B00D29E6E /* Run Crashlytics */ = {
```

Comment:
Does MSAppCenter automatically upload DSYMs? Is there some step we need to take to upload them instead?

Reply:
They have CLi you can install to terminal, but easiest is just drop the zip to the UI

Reply:
Ah no .. we need a script. I have one....